### PR TITLE
added a parser for si2_osc_kurucz atomic data

### PR DESCRIPTION
**Pull request by Chinmay4400**
Completed first objective for project Carsus Atomic Data. Made a module to parse and store atomic data in an hdf file.

## Description
Added a file atomic_data_parse.py containing a function parse_atomic_data to parse and extract atomic data from si2_osc_kurucz file as mentioned in the project objective.
The data is stored in an hdf file as pandas DataFrames.

## How Has This Been Tested?
Added unit test for the function testing the formation of h5 file and checking shapes of resultant DataFrames.

## Screenshots (if appropriate):
Screenshot_20200313_013732

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: scripts/atmomic_data_parse.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    df1.to_hdf('atomic_data.h5',df1.index.name)
+    df2.to_hdf('atomic_data.h5',df2.index.name)
+
+import pandas as pd
+
+parse_atomic_data("si2_osc_kurucz")
+df1 = pd.read_hdf("atomic_data.h5",key="energyLevels")
+df2 = pd.read_hdf("atomic_data.h5",key="oscillator_strengths")
+print(df1.head())
+print(df2.head())
```
Comment (jaladh-singhal) [inline scripts/atmomic_data_parse.py]: Any python module (or script) of a package usually has only functions defined. Executing them or outputting data should be done in a notebook (part of documentation) and in unit tests where you validate returned output against expected output. So you should add test and documentations alongside.

If you just intended to show this as a part of your work-in-progress, you can do that via a gist of notebook executing this.

Comment (Chinmay4400) [reply scripts/atmomic_data_parse.py]: @jaladh-singhal Yes, I intended this to show as a working in progress. I have yet to refine the function and add some comments. I'll update the PR with a notebook and add tests. Thanks!

Comment (Chinmay4400) [reply scripts/atmomic_data_parse.py]: @jaladh-singhal Please take a look at this gist https://gist.github.com/Chinmay4400/f1b3a0c9835573647b5dbbebdfec1cce

Comment (Chinmay4400) [reply scripts/atmomic_data_parse.py]: @jaladh-singhal Should I move this PR to carsus repo ?

**Comment by Chinmay4400** [comment]
The mac test was unsuccessful due to an older image of macOS (as far as I understand) in Azure Pipeline.

File: scripts/atmomic_data_parse.py (hunk 1 above)
Comment (jaladh-singhal) [reply scripts/atmomic_data_parse.py]: @Chinmay4400 yes sure, great that you already did! I was not aware this work s related to carsus.

**Pull request closed without merging**